spyder_vim/vim_widget.py

```diff
@@ -79,14 +89,19 @@ def DOLLAR(self, repeat=1):
     def ZERO(self, repeat=1):
         self._move_cursor(QTextCursor.StartOfLine)
 
-    def G(self, repeat):
-        editor = self._widget.editor()
-        editor.go_to_line(repeat)
-        self._widget.update_vim_cursor()
-
+    def G(self, repeat=1):
+        self._move_cursor(QTextCursor.Start)
```

Comment:
I actually wasn't aware that you could jump lines that way! I thought G and gg were different because I only used their default uses: G goes to last line, gg goes to first line. And actually I mixed their default uses too! Whoops, I'll get on it